packages/polaris-viz-core/src/constants.ts

```diff
@@ -373,7 +373,7 @@ export const LIGHT_THEME: Theme = {
   legend: {
     valueColor: variables.colorGray160,
     labelColor: variables.colorGray100,
-    backgroundColor: variables.colorGray10,
+    backgroundColor: variables.colorGray00,
```

Comment:
Should we allow this to be `null` so it doesn't render any color instead of having to match the background of the chart?

Reply:
Good idea! Done ✅